specification/maps/resource-manager/Microsoft.Maps/preview/2023-12-01-preview/maps-management.json

```diff
@@ -0,0 +1,1517 @@
+{
```

Comment:
There are a bunch of breaking changes reported on this PR. Please get a breaking changes approval from Azure Breaking Changes Reviewers <azbreakchangereview@microsoft.com> and ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.

Reply:
Tags are updated now. Thanks!